miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -179,6 +256,28 @@ def map(self) -> bool:
         """Map token."""
         return bool(self.data["map_present"])
 
+    @property
+    @setting(
+        "Multi map",
+        choices_attribute="multi_map_enum",
+        setter_name="load_multi_map_by_enum",
+        icon="mdi:floor-plan",
+    )
+    def multi_map_id(self) -> int:
+        """The id of the current map with regards to the multi map feature,
+
+        [3,7,11,15] -> [0,1,2,3].
+        """
+        return int((self.data["map_status"] + 1) / 4 - 1)
+
+    @property
+    def multi_map_name(self) -> str:
+        """The name of the current map with regards to the multi map feature."""
+        if self._multi_maps is None:
+            return str(self.multi_map_id)
+
```

Comment:
See my comment above wrt the multi maps. Let's try to keep this PR simple and add support for multi maps in a separate PR :-)